include/openssl/core_names.h

```diff
@@ -143,6 +143,21 @@ extern "C" {
 #define OSSL_KDF_NAME_X963KDF       "X963KDF"
 #define OSSL_KDF_NAME_KBKDF         "KBKDF"
 
+/* RAND parameters */
+#define OSSL_RAND_PARAM_STRENGTH    "strength"                /* unsigned int */
+#define OSSL_RAND_PARAM_SEED        "seed"                    /* octet string */
+#define OSSL_RAND_PARAM_PERSONALISE "personalise"             /* octet string */
+#define OSSL_RAND_PARAM_CIPHER      OSSL_ALG_PARAM_CIPHER     /* utf8 string */
+#define OSSL_RAND_PARAM_DIGEST      OSSL_ALG_PARAM_DIGEST     /* utf8 string */
+#define OSSL_RAND_PARAM_PROPERTIES  OSSL_ALG_PARAM_PROPERTIES /* utf8 string */
+
+/* Known RAND names */
+#define OSSL_RAND_NAME_SEED         "seed"
```

Comment:
Is this for seed source methods?

Reply:
Will there be only one?

Reply:
Yes, this will be for _our_ seeding from the whatever was configured.  I'm happy to rename it, that part isn't done yet & I'm not going finish before I fly out for the F2F.

Reply:
The reason seed sources are included here is because of the way seeding DRBGs works.  A DRBG has a parent DRBG pointer.  If this pointer is set, the parent is used as a seed source.  If the pointer is NULL, the built in source is used instead.  This inhibits having more than one seed source.  EVP_RAND is adding a layer that will allow either to be used for seeding.

Reply:
`ossl-seed`?

Reply:
I'm fine with that.

Reply:
and done.